src/commands/lambda/constants.ts

```diff
@@ -134,3 +134,57 @@ export const SKIP_MASKING_ENV_VARS = new Set([
   SITE_ENV_VAR,
   VERSION_ENV_VAR,
 ])
+
+// Project files to search for in Flare
+export const PROJECT_FILES = [
+  // Serverless framework
+  'serverless.yml',
+  'serverless.js',
+  // AWS CDK
+  'cdk.json',
+  // CloudFormation Template
+  'template.yml',
+  'template.json',
+  'datadog-cloudfomation-macro.yml',
+  'datadog-cloudformation-macro.json',
+  // Node.js
+  'package.json',
+  'package-lock.json',
+  'yarn.lock',
+  '.nvmrc',
+  // Python
+  'requirements.txt',
+  'Pipfile',
+  'Pipfile.lock',
+  'pyproject.toml',
+  // Java
+  'pom.xml',
+  'build.gradle',
+  'gradlew',
+  'gradlew.bat',
+  // Go
+  'Makefile',
+  'go.mod',
+  'go.sum',
+  'Gopkg.toml',
+  'gomod.sh',
+  // Ruby
+  'Gemfile',
+  'Gemfile.lock',
+  // .NET
+  'project.json',
+  'packages.config',
+  'PackageReference',
+  'global.json',
+  'AwsDotnetCsharp.csproj',
+  // Docker
+  'Dockerfile',
+  'docker-compose.yml',
+  // Webpack, bundlers
+  'webpack.config.js',
+  'rollup.config.js',
```

Comment:
What is `rollup.config.js`?

Reply:
It's a config file for Rollup, a JavaScript bundler. I figured it wouldn't hurt to include, WDYT?

Reply:
I haven't heard of any customer using `rollup.config.js` yet, and we also don't offer any documentation around using this bundler and our products. Maybe we should remove it until that comes?